module/icp/algs/impl/impl.c

```diff
@@ -0,0 +1,356 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+ * or http://www.opensolaris.org/os/licensing.
+ * See the License for the specific language governing permissions
+ * and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+/*
+ * Copyright 2021 Jan Kasiak. All rights reserved.
+ */
+
+/*
+ * Based on aes_impl.c
+ */
+
+#include <sys/simd.h>
+#include <impl/impl.h>
+
+static const struct {
+	char *name;
```

Comment:
`const char *name`

Reply:
Fixed.

Reply:
I forgot to push out my commit. Should be resolved now :D